app/components/GettingStartedModal/steps/SummaryStep/SummaryStep.tsx

```diff
@@ -58,11 +62,22 @@ const SummaryStep = ({
             blueprintsInstallationTasks.tasks
         ) {
             let installationFinished = false;
+            // eslint-disable-next-line @typescript-eslint/no-shadow
+            let installationStatuses = {} as Record<TaskType, Record<string, TaskStatus>>;
             const resourcesInstaller = createResourcesInstaller(
                 manager,
                 internal,
                 () => handleInstallationStarted(),
-                (progress: number) => setInstallationProgress(progress),
+                (progress: number, taskType?: TaskType, taskName?: string, taskStatus?: TaskStatus) => {
+                    if (taskType != null && taskName != null && taskStatus != null) {
```

Comment:
Since none of these values can be `null` I think it would be less confusing (I'm assuming `taskName` will never be `''`):
```suggestion
                    if (taskType && taskName && taskStatus) {
```

Reply:
done